### DISCUSSION TITLE
Quick Actions

### DISCUSSION THREAD

**Discussion by Rigido71** [StigNygaard/xIFr#20] [General]
Hi Stig,
and thank you for this extension that just does what it says...and it does it well! 
I know it may look not "polite" but...do you think it would be possible to add some "quick actions" (buttons, icons, whatever) to:
1. copy to clipboard the link to the image
2. copy to clipboard the image itself
3. save the image

To make somehow this post useful, when the name of the image is too long, it overlaps the xIFr icon:
<img width="94" alt="image" src="https://github.com/StigNygaard/xIFr/assets/29772377/7a70d104-9c8d-47fc-a856-18a48ccdbf1d">

Maybe you can move the icon some pixels down. 😄 

Grazie!

**Comment by StigNygaard** [comment]
Hi @Rigido71 

Oh, I think it looks like a polite suggestion, and polite suggestions are always welcome (though sometimes they also are politely rejected) 🙂😉

The first one would definitely be easy to implement. And I can see it could be a help in some situations, but it is not that much harder to right-click link and choose "copy" - or click the link and copy url from browser's address-bar?

But I'm probably not going to implement these suggestions. Though I see there are lot of "perfectly legal reasons" to make use of such a features, I fear it will let xIFr look like a tool dedicated to steal images. And as I care a lot about rights and licensing, I don't want xIFr to look like that.

Yeah, I've been too lazy to fix that icon issue. But I think icon looks best in corner, so if I fix the overlap, it will probably be by cutting text shorter. Maybe...

Kind regards
Stig

**Comment by Rigido71** [comment]
Thank you Stig, 
you perfectly pinned the point.
I'll wait for the new version that fix the icon issue...maybe 😂

Thank you
Andrea.
